scripts/utils.py

```diff
@@ -70,3 +68,12 @@ def get_secret(secret_name: str) -> None:
         return json.loads(get_secret_value_response["SecretString"])
     else:
         return json.loads(base64.b64decode(get_secret_value_response["SecretBinary"]))
+
+
+def get_sf_ingestion_arn():
+    sts = boto3.client("sts")
+    ACCOUNT_ID = sts.get_caller_identity().get("Account")
+    REGION = os.environ.get("AWS_REGION", "us-east-1")
```

Comment:
Are all of our resources in `us-west-2`? Might be able to change the default value

Reply:
Fixed.